ac/ac-quiz/src/ActivityRunner/Quiz.js

```diff
@@ -36,6 +36,7 @@ const BottomQuizNav = withStyles(styles)(
     onSubmit,
     hasNext,
     hasAnswered,
+    allowSkip,
```

Comment:
The `onSubmit` will not work until all questions are answered. MAybe you want to check for `allowSkip` there too 

https://github.com/chili-epfl/FROG/pull/1324/commits/3f08c35753b148110cd2882eb57886f2f92fea4b#diff-c06287971b4bbb6312864b537e98f6feR111